src/app/checkout/accordion.controller.js

```diff
@@ -650,7 +572,62 @@ angular.module('checkoutModule')
             });
         }
 
-        function applyDiscount() {
+        $scope._sendPostForm = function(method, response) {
```

Comment:
maybe rename `_sendPostForm` to something more descriptive of the action we are taking on a Discount?